lava-master/Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM baylibre/lava-master-base:2019.03_stretch
+FROM lavasoftware/lava-server:2019.07
+
+RUN apt-get update
+RUN apt-get -y install lava-dispatcher sudo
```

Comment:
Don't you need the `apt-get update` and `apt-get install` on the same line to avoid docker caching issues?

Reply:
OK